### DISCUSSION TITLE
Blog post and related assets in same folder

### DISCUSSION THREAD

**Discussion by florianwachs** [statiqdev/Discussions#127] [Q&A] [answered]
Hi there, I want build a personal site with statiq. All examples I found use a blog folder and a separate assets folder for all content. I would much more prefer to have a folder per blog post and the assets relative to that.
so:
```
blog
      |- Csharp10features
                                    |- New_features_in_Csharp_10.md
                                    |- asset1.jpg
                                    |- asset2.svg


```

Is that possible with statiq?

**Comment by ror3d** [comment, marked as answer]
Yes, in general any asset can or will be copied to the output as is, unless it's processed.

If you find that the directory they end up in is not the one you want them to be, you can use [directory metadata](https://www.statiq.dev/web/content-and-data/metadata/directory-metadata) or sidecar files to change where they are copied to, with something like

    DestinationPath: => $"{Document.Destination.FileName}"

**Reply by florianwachs** [reply]
Thank you, will try that!
